mlflow/pipelines/steps/train.py

```diff
@@ -330,6 +331,7 @@ def my_warn(*args, **kwargs):
                     "training": train_df,
                     "validation": validation_df,
                 }.items():
+                    metric_prefix = f"{dataset_name}_"
```

Comment:
This should be `train_`, `val_`, instead of `training_`, `validation_`, right?

Reply:
Can we also update train step test coverage and add `val_` to the train step test coverage?

Reply:
Thanks for the catch! `validation_` should be changed to `val_`, but not sure about `training_`.

Reply:
Sklearn autologging uses `training_`:

https://github.com/mlflow/mlflow/blob/d11299ef2622810ba96e2049b183fee37aae31a2/mlflow/sklearn/utils.py#L19

Reply:
Thanks @harupy ! Didn't realize AutoML used `training_` instead of `train_`. Changing to `val_` sounds good :)